docusaurus/docs/tutorials/build-a-streaming-data-source-plugin.md

```diff
@@ -291,21 +291,21 @@ However, there is an easier way of doing all this.
 npm run server
 ```
 
-This command runs a Grafana container using docker-compose and puts the built plugins in the right place.
+This command runs a Grafana container using docker compose and puts the built plugins in the right place.
 
 2. To verify the build, go to Grafana at `https://localhost:3000`.
 
 :::note
 
-You may be using Docker Compose instead of docker-compose. If that is your case, you just need to change the command in the `package.json` in the project's root directory.
+You may be using Docker Compose instead of docker compose. If that is your case, you just need to change the command in the `package.json` in the project's root directory.
```

Comment:
Should we delete this sentence? I don't think it makes any sense.

Reply:
Good catch, removed https://github.com/grafana/plugin-tools/pull/1064/commits/d0989c3e6b0ef324ac6fac12cf90c8d9a2c14eda